packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx

```diff
@@ -107,31 +107,31 @@ export class PersonaCoinBase extends BaseComponent<IPersonaCoinProps, IPersonaSt
     return (
       <div {...divProps} className={classNames.coin}>
         {// Render PersonaCoin if size is not size10
-        size !== PersonaSize.size10 && size !== PersonaSize.tiny ? (
-          <div {...coinProps} className={classNames.imageArea} style={coinSizeStyle}>
-            {!this.state.isImageLoaded &&
-              (!imageUrl || this.state.isImageError) && (
-                <div
-                  className={mergeStyles(
-                    classNames.initials,
-                    !showUnknownPersonaCoin && { backgroundColor: initialsColorPropToColorCode(this.props) }
-                  )}
-                  style={coinSizeStyle}
-                  aria-hidden="true"
-                >
-                  {onRenderInitials(this.props, this._onRenderInitials)}
-                </div>
+          size !== PersonaSize.size10 && size !== PersonaSize.tiny ? (
+            <div {...coinProps} className={classNames.imageArea} style={coinSizeStyle}>
+              {onRenderCoin === this._onRenderCoin && !this.state.isImageLoaded &&
+                (!imageUrl || this.state.isImageError) && (
```

Comment:
Could we pull this logic out into a descriptive `const`? It's becomes quite a difficult piece to understand by itself like this